### PR TITLE
Baidu

### PR DESCRIPTION
**Pull request by Madis0**

### CODE REVIEW

File: lite-apps/index.json [hunk 1]
```diff
@@ -91,7 +91,10 @@
       },
       {
         "name": "Bing"
-      }
+      },
+	  {
```
Comment (chimbori-dev) [inline lite-apps/index.json]: Please indent according to the rest of the file; we will add a pre-submit step that will do this automatically.

File: lite-apps/Baidu/manifest.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  "hermit_settings": {
+    "open_links": "browser"
+  },
+  "hermit_search": [
+    {
+      "url": "http://www.baidu.com/s?wd=%s",
+      "name": "__MSG_search__"
+    }
+  ],
+  "hermit_share": [
```
Comment (chimbori-dev) [inline lite-apps/Baidu/manifest.json]: Please revert the Share integration for Search.

File: lite-apps/Baidu/manifest.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  "start_url": "https://baidu.com",
+  "manifest_url": "https://hermit.chimbori.com/lite-apps/Baidu.hermit",
+  "secondary_color": "#a1b4e5",
+  "lang": "en",
+  "icons": [
+    {
+      "src": "icon.png"
+    }
+  ],
+  "hermit_settings": {
```
Comment (chimbori-dev) [inline lite-apps/Baidu/manifest.json]: Let’s leave these settings as default.

**Comment by chimbori-dev** [comment]
Ouch, I accepted and merged the first pull request, but now the other two have conflicts. :(

**Comment by Madis0** [comment]
What if you just force merge them all and fix index.json afterwards?

**Pull request merged**